benchmark/http/_http_simple.js

```diff
@@ -4,10 +4,10 @@ var http = require('http');
 
 var port = parseInt(process.env.PORT || 8000);
 
-var fixed = 'C'.repeat(20 * 1024),
-  storedBytes = {},
-  storedBuffer = {},
-  storedUnicode = {};
+var fixed = 'C'.repeat(20 * 1024);
+var storedBytes = Object.create(null);
+var storedBuffer = Object.create(null);
+var storedUnicode = Object.create(null);
```

Comment:
> AFAIK `Map` is still slow and http benchmarks already take a very long time as it is.

That is why we have https://github.com/nodejs/node/blob/master/benchmark/es/map-bench.js  - tl;dr, it's faster.